src/plugins/sensor/ContactBlobCamera/ContactBlobCamera.cpp

```diff
@@ -367,6 +375,14 @@ void ContactBlobCamera::add_false_positives(
 bool ContactBlobCamera::step() {
     if ((time_->t() - last_frame_t_) < 1.0 / fps_) return true;
 
+    if (proj_.LatitudeOrigin() != parent_->projection()->LatitudeOrigin() ||
```

Comment:
I'm not sure this is a scalable solution since it sort of breaks the idea that a scrimmage simulation takes place in a single ENU frame. I'm fine with translations to external systems that use lat/lon/alt, but a scrimmage plugin shouldn't be changing frames. Is there a way that you can manage this externally?

Reply:
If it makes a difference to you, this plugin does not ever change the projection, it only monitors for a change and recomputes targets when it does.